redcap_bridge/test_redcap/testfiles/elabConversion/csvRecord.csv

```diff
@@ -0,0 +1,52 @@
+Variable / Field Name,Form Name,Section Header,Field Type,Field Label,"Choices, Calculations, OR Slider Labels",Field Note,Text Validation Type OR Show Slider Number,Text Validation Min,Text Validation Max,Identifier?,Branching Logic (Show field only if...),Required Field?,Custom Alignment,Question Number (surveys only),Matrix Group Name,Matrix Ranking?,Field Annotation
```

Comment:
I think for a start it would be good to work on a simplified version of this file, that only contains one (or a few) rows for each Field Type we need to cover.

Reply:
Yes, that's what I think too. I was just simplifying the csv

Reply:
This would be a good next step to move forward with adding systematic testing

Reply:
Test each sub-function to see if it returns the correct dictionary?

Reply:
I think thats a bit too fine grained. I would rather do the testing of all of them based on the the small test file example, as described here: https://github.com/INT-NIT/DigLabTools/pull/59#issuecomment-1301925648